cinema_service/urls.py

```diff
@@ -1,9 +1,12 @@
+from django.conf.urls.static import static
```

Comment:
Code Style: Group imports using `()` if needed. Consider grouping the imports from `django` together for better readability.